CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## Fixed
+- Updated links in the readme to point to the correct TinyMCE 7 documentation. #INT-3300
```

Comment:
```suggestion
- Updated links in the readme to point to the latest TinyMCE documentation. #INT-3300
```

Reply:
I don't know if we even need a changelog for this 🤷‍♂️

Reply:
I've removed the changelog entry